doc/appendix-de-kassensichv/cash-register-integration/receipt-sequences-creation.md

```diff
@@ -0,0 +1,174 @@
+
+#### Workflow example
+
+![chargeitem-internal-payment-external](media/chargeitem-internal-payment-external.png)
+
+### Option B: ChargeItems collected at an external system or queue are payed at the internal queue
+
+#### How to use
+
+Use 'info-internal' with ['ftReceiptCaseData' according to the requirements of the DSFinV-K-specification](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/dsfinv-k#file-bon_referenzen-referencescsv) to reference to an business-action recorded by an external queue or POS system which needs to be merged with your ongoing internal business-action. By creating a new 'cbReceiptReference', you create the precondition to merge the receipt the external system with your internal receipts of the ongoing business-action. Repeat this step to collect and reference to multiple external POS systems with related business-actions to be charged. The obligation to issue receipts arises at the POS system where the POS receipt is being created.
+
+##### Prerequisites
+
+For this workflow, the combination of following receipt-sequences is needed:
+
+- Referencing previous receipts within a queue, using 'cbReceiptReference',
+- Merging receipts, using 'cbReceiptreference' and 'cbPreviousReceiptReference',
+- Providing additional information about how the business actions are connected to each other, using 'cbArea',
+- Providing information about the external POS receipt, using 'ftReceiptCaseData'
+
+#### Workflow example
+
+![chargeitem-external-payment-internal](media/chargeitem-external-payment-internal.png) 
+
+1. A couple performs a check-in at the reception of a hotel for one night.
+2. An info-order for the overnight-stay is created.
+3. After the check-in, it decides to have a beer at the hotel-bar, which uses a different POS system (or fiskaltrust.queue). The consumption of the hotel-bar shall be paid with the final invoice of the overnight-stay. The room number is for 'cbArea' to provide information why the business actions across the different POS systems are connected. 
+4. For the check-out, the receipt of the consumption of the hotel-bar and the receipt of the overnight stay need to be merged. Therefore, 'info-internal' receipts with a new, common 'cbReceiptReference' are created. One 'info-internal' receipt is used to reference to the external POS receipt using 'ftReceiptCaseData'. 
+5. The other 'info-internal' receipt is used to reference to the internal 'info-order' of the overnight-stay using 'cbPreviousReceiptReference'. 
+6. A 'POS receipt' is created, including all collected charge-items from external and internal POS system(s), and the pay-items of the internal POS system. The receipt is printed and handed over to the couple.
+
+#### Code examples
+
+Code examples of referencing external receipts can be found in our postman collection.
```

Comment:
Link is missing here

Reply:
yes, because i don't know it yet. has to be published first. i wrote some tasks in our pbi to not forget that..:)